__tests__/app.test.js

```diff
@@ -44,3 +44,30 @@ describe("GET /api/", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  it("200: should respond with an article with the correct id and all expected properties", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
This endpoint is expecting an article object, whereas this is an object with a single value array containing the article
Is there a better point in the code for the article to be taken out of the array?